### PR TITLE
Date: December 8th Yanice commit

### PR DESCRIPTION
**Pull request by YaniceSze**
Footer CoreValues TeamSection and FAQ added.

### CODE REVIEW

File: src/components/ui/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import SocialMediaIcons from './socialMedia';
+
+const Footer: React.FC = () => {
+  return (
+    <footer className="bg-black text-white py-6">
```
Comment (ssaini4) [inline src/components/ui/footer.tsx:6]: background here is `#202020`

File: src/components/ui/coreValues.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <div className="max-w-6xl mx-auto mt-10">
+      {values.map((value, index) => (
+        <div
+          key={index}
+          className="flex items-start py-6 border-b border-gray-300 last:border-0 gap-6"
+        >
+          {/* Folder Icon */}
+          <div className="flex-shrink-0 w-20">
+            <FolderIcon className="text-gray-500" fontSize="large" />
```
Comment (ssaini4) [inline src/components/ui/coreValues.tsx:17]: this icon needs to be the outline Folder icon. See:
Screenshot 2024-12-10 at 7 08 35 AM

File: src/components/ui/socialMedia.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <div className="flex justify-center space-x-6">
+      {/* Facebook Icon */}
+      <a
+        href="https://facebook.com"
+        target="_blank"
+        rel="noopener noreferrer"
+        className="text-gray-400 hover:text-white transition-colors duration-200"
+      >
+        <FacebookIcon/>
+      </a>
```
Comment (ssaini4) [inline src/components/ui/socialMedia.tsx:17]: These icons need to have a circle background. See

Screenshot 2024-12-10 at 7 09 42 AM

File: src/components/ui/team.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    key={memberIndex}
+                    className="w-full sm:w-1/2 lg:w-1/3 px-4 mb-12"
+                  >
+                    <div className="max-w-[250px] mx-auto font-default">
+                      <Image
+                        src={member.image}
+                        alt={member.name}
+                        width={150}
+                        height={150}
+                        className="block mb-6 rounded-full object-cover"
```
Comment (ssaini4) [inline src/components/ui/team.tsx:35]: These images don't need to be rounded and should be bigger. See:
Screenshot 2024-12-10 at 7 11 01 AM

**Review by ssaini4 — CHANGES_REQUESTED** [review]

**Comment by YaniceSze** [comment]
Changes implemented but didn't find the correct icon for the facebook. The one I found is only for the paid version.

**Comment by ssaini4** [comment]
Thank you for the fixes and address the comments @YaniceSze. This looks good to me! 👍

**Pull request merged**